demo/src/index.js

```diff
@@ -8,6 +8,7 @@ import './index.scss'
 
 // Configuration
 let ConfigDefaults = {
+	site_title: "TeskaLabs",
```

Comment:
The default should be empty.

Reply:
actually I think it should be `site_title: "ASAB App"` and `title` should be empty (since the title has format `site_title | title`), correct?

Reply:
Format is to be either:

`title` or `site_title | title`